include/swift/Basic/Features.def

```diff
@@ -408,6 +408,9 @@ SUPPRESSIBLE_EXPERIMENTAL_FEATURE(AllowUnsafeAttribute, true)
 /// Warn on use of unsafe constructs.
 EXPERIMENTAL_FEATURE(WarnUnsafe, true)
 
+// Enable values in generic signatures, e.g. <let N: Int>
+EXPERIMENTAL_FEATURE(ValueGenerics, true)
```

Comment:
Should the feature flag name be more specific, like `IntGenerics`, in case we want to use `ValueGenerics` as a feature flag for a more general feature in the future?

Reply:
Sure, we could name it that if you'd like. We could also do `ValueGenerics2` or something in the future (or `ValueGenericsMoreBetter`)